res/sections/20-Lez06032018.tex

```diff
@@ -1,16 +1,17 @@
 \chapter{Start-up}
 
-Che cos'è una \textit{startup}? Generalmente si un'azienda giovane creata da
-profili giovani. Di conseguenza tende ad avere una visione più ``giovane'' di
-lavoro e azienda. In questo contesto in 2-3 anni di esperienza un programmatore
-non è più \textit{junior}, ma considerato già \textit{mid} o addirittura
-\textit{senior}. In aziende più strutturate invece il tempo per abbandonare la
-categoria \textit{junior} è molto più lungo e varia dai 5 ai 10 anni (in alcuni
-contesti addirittura 15). Soprattutto quando si è in questa categoria è
-necessario saper ascoltare i programmatori più esperti e capire le dinamiche
-interne di un'azienda, facendo attenzione nel caso in cui si vogliano proporre
-nuove idee, ancora di più nel caso in cui non si è stati assunti per portare
-qualcosa di ``nuovo''. Le idee proposte potrebbero essere considerate infatti:
+Che cos'è una \textit{startup}? Generalmente si intende un'azienda giovane
+creata da profili giovani. Di conseguenza tende ad avere una visione più
+``giovane'' di lavoro e azienda. In questo contesto in 2-3 anni di esperienza
```

Comment:
Ci son 3 ripetizioni di `giovane` in 2 frasi :sweat_smile:  Per il momento è ok, in futuro qualcuno metterà a posto